E/extrae/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env julia
+using BinaryBuilder, Pkg
+
+name = "extrae"
+version = v"4.0.1"
+sources = [
+    ArchiveSource(
+        "https://github.com/bsc-performance-tools/extrae/archive/refs/tags/$(version).tar.gz",
+        "e6765eb087be3f3c162e08d65f425de5f26912811392527d56ecd75d4fb6b99d"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/extrae-*
+autoreconf -fvi
+./configure \
+    --prefix=$prefix \
+    --with-mpi=$prefix \
+    --without-binutils \
+    --without-dyninst \
+    --without-papi \
+    --without-unwind
+
+make -j${nproc}
+make install
+"""
+
+platforms = expand_cxxstring_abis([
+    Platform("i686", "Linux"),
+    Platform("x86_64", "Linux"),
+])
+
+products = [
+    LibraryProduct("libseqtrace", :libseqtrace),
+    LibraryProduct("libomptrace", :libomptrace),
+    LibraryProduct("libnanostrace", :libnanostrace),
+]
+
+dependencies = [
+    Dependency("OpenMPI_jll"),
```

Comment:
I think it'd be nicer to target more MPI implementations, you can look at https://github.com/JuliaPackaging/Yggdrasil/blob/6a875bb6bdfbd8c299118fec79f1ce0b85cc6b63/A/ADIOS2/build_tarballs.jl#L81-L97 for inspiration

Reply:
Thanks!